ddtrace.gemspec

```diff
@@ -69,5 +69,8 @@ Gem::Specification.new do |spec|
   # Used by profiling (and possibly others in the future)
   spec.add_dependency 'libdatadog', '~> 0.7.0.1.1'
 
+  # Used by method tracing API
+  spec.add_dependency 'datadog-instrumentation'
```

Comment:
Because this package is not published to RubyGems we can't merge this with this dependency as-is.

Eventually we should publish this, but in the interim, we could introduce it as a soft dependency: remove it from this file, and add a check to `trace_method` that looks for the presence of `Datadog::Instrumentation`.

 - If present, use the package and proceed as normal.
 - If not present, log a warning and suggestion to add `datadog-instrumentation` to the `Gemfile`.

This way, users could opt-in to use this as a "beta" feature while we polish up `datadog-instrumentation` for it's debut, without causing errors.